jujugui/static/gui/src/app/components/user-profile-header/user-profile-header.js

```diff
@@ -0,0 +1,31 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('user-profile-header', function() {
+
+  juju.components.UserProfileHeader = React.createClass({
+
+    render: function () {
+      return (<div className="user-profile-header"></div>);
```

Comment:
Yeah we're not really sure how to get the data necessary to do this so this is just a placeholder for now.